app/_landing_pages/gateway/kong-manager.yaml

```
@@ -0,0 +1,147 @@
+metadata:
+  title: Kong Manager
+  product: gateway
+  content_type: landing_page
+  description: Kong Manager is the graphical user interface (GUI) for {{site.base_gateway}}.
+rows:
+  - header:
+      type: h1
+      text: "Kong Manager"
+
+  - columns:
+      - blocks:
+          - type: structured_text
+            config:
+              header:
+                text: "What is Kong Manager?"
+              blocks:
+                - type: text
+                  text: | 
+                    Kong Manager is the graphical user interface (GUI) for {{site.base_gateway}}.
+                    It uses the [Admin API](/api/gateway/admin-ee/) under the hood to administer and control {{site.base_gateway}}. 
+                    If you're running {{site.base_gateway}} on-prem with a database (either in traditional or hybrid mode), you can enable Kong Manager.
+
+                    You can do the following with Kong Manager:
+                - type: unordered_list
+                  items:
+                    - Manage all Workspaces in one place
+                    - Create and manage Routes and Gateway Services
+                    - Activate or deactivate plugins
+                    - Manage Certificates
+                    - Manage users and roles for {{site.base_gateway}}
+                    - Centrally store and easily access Key Sets and Keys
+                    - Manage Vaults
+      - blocks:
+          - type: image
+            config:
+              url: /assets/images/gateway/kong-manager-ui.png
+              alt_text: Image of the Kong Manager UI
+  
+  - columns:
+      - blocks:
+        - type: structured_text
+          config:
+            header:
+              text: "Access Kong Manager"
+            blocks:
+              - type: text
+                text: |
+                  If you're running {{site.base_gateway}} on-prem with a database (either in traditional
+                  or hybrid mode), you can [enable Kong Manager](/gateway/enable-kong-manager/). If you run the [{{site.base_gateway}} quickstart script](https://get.konghq.com/quickstart), Kong Manager is automatically enabled.
+
+                  If you want to enable Kong Manager for multiple domains in {{site.base_gateway}} 3.9.x or later, see [Multiple domains](/gateway/enable-kong-manager/#multiple-domains).
+                  
+                  To access Kong Manager, go to the following URL: [http://localhost:8002](http://localhost:8002)
+  
+  - columns:
+      - blocks:
+        - type: structured_text
+          config:
+            header:
+              text: "Kong Manager authentication"
+            blocks:
+              - type: text
+                text: |
+                  {{site.base_gateway}} comes packaged with authentication plugins that can be used to secure Kong Manager.
```

```suggestion
                  Kong Manager comes packaged with several authentication methods.
```
While they are sort of using the Kong plugins in the backend, you don't configure Kong plugins to secure Kong Manager. It's a weird tangential config method that, to the user, is only applicable in the Kong Manager context.